src/templates/rpc/Dockerfile_rpc_dev

```diff
@@ -1,11 +1,14 @@
 # Use an official Python runtime as the base image
-FROM python:3.11-slim
+FROM python:3.12-slim
 
 # Install procps, which includes pgrep
 RUN apt-get update && \
     apt-get install -y procps openssh-client && \
     rm -rf /var/lib/apt/lists/*
 
+# Install `uv` packge installer (https://github.com/astral-sh/uv)
```

Comment:
spelling: package

Reply:
done and installed spellchecker for markdown files :P